RevisionHistory.html

```diff
@@ -325,3 +325,20 @@
 <li>Added more help info to user drawn profiles</li>
 </ul></ul>
 </ul>
+
+<ul><li>Version 6.3.1</li>
+<ul>
+<li>Fixed some issues with regions that caused DFTF to crash if you didn't check them each time\</li>
+<li>Regions are remembered better from one igram to the next</li>
+<li>Fixed a few minor bugs related to "edge mask":
+<ul>
+<li>null calculation when first opening DFTF</li>
+<li>saving edge mask</li>
+<li>improved an edge mask message wording</li>
+</ul>
+</li>
+<li>Improved many things that only affect DFTFringe programming team</li>
+<li>New windows installer</li>
+<li>Linux version of DFTF now exists (only for latest ubuntu) and is always built automatically</li>
```

Comment:
This is only a suggestion. I'm not native English speaker. Take what you think is best.

As I said in email :
```suggestion
<li>DFTF can now build and run on Linux (build is only tested on latest Ubuntu)</li>
<li>Move to semver format for version names</li>
```